database/index.js

```diff
@@ -0,0 +1,22 @@
+const mongoose = require('mongoose');
+const Transaction = require('./Transaction.js');
+const Stock = require('./Stock.js');
+
+// connect to mongo via localhost
+var promise = mongoose.connect('mongodb://localhost/robinsHood', {
+  useMongoClient: true
+});
+
+// define connection
+const db = mongoose.connection;
+
+// if there's a connection error log it
+db.on('error', console.error.bind(console, 'connection error:'));
+
+// RUN THIS CODE TO SEED MONGO DATABASE
+// mongoimport --db robinsHood --collection stocks --file nasdaq-data.csv --type csv --headerline
```

Comment:
Are you only running this code in your terminal? So it just auto seeds it into your mongo db?

Reply:
Yep, just csv terminal import